file_root/generics/openstack_packages.sls

```diff
@@ -5,4 +5,22 @@ openstack_selinux_install:
     - name: {{ salt['pillar.get']('packages:openstack_selinux') }}
     - require:
       - pkg: juno_repo_install
+
+network_manager_dead:
+  service:
+    - dead
+    - name: {{ salt['pillar.get']('services:network_manager') }}
+    - enable: False
+
+network_manager_removed:
+  pkg:
+    - purged
+    - name: {{ salt['pillar.get']('services:network_manager') }}
+    - require:
+      - service: network_manager_dead
+
+network_service_enabled:
```

Comment:
Should be specified as a requirement prior to installation, switching from NM to network service without a proper configuration might lead to loosing connectivity.